logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/asset/ScheduleNewAssetMessageUseCase.kt

```diff
@@ -125,68 +136,138 @@ internal class ScheduleNewAssetMessageUseCaseImpl(
                 senderClientId = currentClientId,
                 status = Message.Status.PENDING,
                 editStatus = Message.EditStatus.NotEdited,
-                expectsReadConfirmation = expectsReadConfirmation
```

Comment:
Asset messages don't expect read confirmation? Not so well versed about this feature but maybe it would be good if @vitorhugods could also chip in about this?

Reply:
If this is not needed, then we should also remove the `userPropertyRepository` from the constructor.